src/index.html

```diff
@@ -1,22 +1,446 @@
+                    <img
+                      src="./images/portable-speakers-a.png"
+                      alt="Portable Speakers photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/portable-speakers-b.png"
+                      alt="Portable Speakers photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Portable speakers</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-a.png"
+                      alt="Earphones photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-b.png"
+                      alt="Earphones photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Earphones</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-A.png"
+                      alt="Smart Home photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-b.png"
+                      alt="Smart Home photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+
+              <div class="category__button">
+                <button class="button">
+                  <a
+                    href="#"
+                    class="button__link"
+                  >
+                    View all
+                  </a>
+                </button>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section
+          class="about-us"
+          id="about-us"
+        >
+          <div class="about-us__banner"></div>
```

Comment:
this banner should fill the hole width of the page. right now it respected the padding of the page. you should remove this section from the section that contain the padding logic and after this section add back this padding value